Yoast/wordpress-seo
# Noindex XMLRPC output

## Summary


This PR can be summarized in the following changelog entry:

* The `xmlrpc.php` file and all possible ways to request it are now automatically `noindex`'d, removing them from Google's search results.

## Relevant technical choices:

* Didn't unittest the header output as that's proving impossible.

## Test instructions

This PR can be tested by following these steps:

* Open example.com/xmlrpc.php, see the HTTP header output contains an `X-Robots-Tag: noindex, follow` header.

## Quality assurance


Fixes PROD-191
